benchmark/src/main/java/com/amazon/opendistroforelasticsearch/sql/benchmark/utils/load/mysql/MysqlDataTransformer.java

```diff
@@ -15,14 +15,25 @@
 
 package com.amazon.opendistroforelasticsearch.sql.benchmark.utils.load.mysql;
 
+import com.amazon.opendistroforelasticsearch.sql.benchmark.utils.CommandExecution;
 import com.amazon.opendistroforelasticsearch.sql.benchmark.utils.load.DataFormat;
 import com.amazon.opendistroforelasticsearch.sql.benchmark.utils.load.DataTransformer;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.util.Arrays;
+import java.util.List;
 
 /**
  * Data transformer for MySQL database.
  */
 public class MysqlDataTransformer implements DataTransformer {
 
+  private String transformedDataPath;
```

Comment:
Why is this a variable - seems like it's only used within transformData?

Reply:
In starting I thought I might need it for the uploader. Forgot to change. removed.